apps/main-landing/src/app/providers/query.tsx

```diff
@@ -0,0 +1,24 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
```

Comment:
not sure about placing this inside `app`, if you put page.tsx inside providers you will expose `idriss.xyz/providers` page which is wrong, can we move it to `src/providers` ?